sawtooth-daml-rpc/src/main/java/com/blockchaintp/sawtooth/daml/rpc/SawtoothReadService.java

```diff
@@ -53,7 +51,7 @@
 
   private static final int DEFAULT_MAX_CLOCK_SKEW = 40; //2x the TimeKeeper period
 
-  private static final Timestamp BEGINNING_OF_EPOCH = new Timestamp(0);
+  private static final Timestamp BEGINNING_OF_EPOCH = Time.Timestamp$.MODULE$.assertFromLong(0);
```

Comment:
This seems weird for a constant, is it strictly necessary? 

Also we've been trying to avoid referring to any of the scala $ constructs since they can cause a bit of confusion when brought into java context where $ generally is used to name inner classes etc.  So again is it strictly necessary?

Reply:
the class has been made package private, so you need to access the Object, which has this Name$.MODULE$ syntax when called from java.

will do a bit more digging to see if there is a nicer way to do this, and also why the class was made private if it was not previously

Reply:
i went back to the old syntax and although the IDE complains it seems to compile